backend/controllers/advertisementController.ts

```diff
@@ -3,16 +3,51 @@ import { AdvertisementApplicationService } from '../application/advertisement'
 
 export class AdvertisementController {
   private advertisementApplicationServise: AdvertisementApplicationService
-  constructor(){
+  constructor() {
     this.advertisementApplicationServise = new AdvertisementApplicationService()
   }
   async createAdvertisement(req: express.Request, res: express.Response) {
     try {
       const advertisementData = req.body
-      await this.advertisementApplicationServise.createAdvertisement(advertisementData)
+      await this.advertisementApplicationServise.createAdvertisement(
+        advertisementData,
+      )
```

Comment:
```suggestion
      await this.advertisementApplicationService.createAdvertisement(
        advertisementData,
      )
```

serviceに直しておいて欲しいです！
→一つ前のPRで気づきませんでした。。。

Reply:
全体的に綴りは修正お願いします。